### ISSUE TITLE
unable to query for a xdmcp session

### ISSUE THREAD

**Issue by frasty** [pelya/xserver-xsdl#80]
Hi,
I'm trying to query a local xdm server for a xdmcp session.
I added "XSDL -query localhost" as a command line parameter and it doesn't work (any parameter I put there doesn't seem to get evaluated, though).
I also changed the data installation location via "specify directory" as suggested in issue 63 with no luck.
Any thoughs?
thank you!

**Comment by pelya** [comment]
Did you modify xdm config files, as described in Play Store app description?

**Comment by frasty** [comment]
> Did you modify xdm config files, as described in Play Store app description?

Yes I followed the instructions on the play store, but unfortunately it doesn't work.
I also tried to modify the parameters and typed "XSDL -nolisten tcp" just to see if it would stop to listen to port 6000, but it didn't. It is still listening to that port:
```
# netstat -lpn|grep 6000
tcp        0      0 0.0.0.0:6000            0.0.0.0:*               LISTEN      26376/x.org.server
```
